Sources/Networking/Responses/PaywallComponentsData.swift

```diff
@@ -32,13 +32,16 @@ public struct PaywallComponentsData: Codable, Equatable, Sendable {
 
         public var stack: PaywallComponent.StackComponent
         public let stickyFooter: PaywallComponent.StickyFooterComponent?
+        public var background: PaywallComponent.Background
 
         public init(
             stack: PaywallComponent.StackComponent,
-            stickyFooter: PaywallComponent.StickyFooterComponent?
+            stickyFooter: PaywallComponent.StickyFooterComponent?,
+            background: PaywallComponent.Background
         ) {
             self.stack = stack
             self.stickyFooter = stickyFooter
+            self.background = background
```

Comment:
Background is required on paywall. We _could_ default to nothing mostly safely but it might show some weird and unwanted results between light and dark mode

Reply:
I think this makes a lot of sense from an SDK perspective. The dashboard can probably set a sensible default for the background, right? (Both for light and dark mode.)

Reply:
Correct correct! Dashboard will make senseable defaults 😊